proxy/envoy/testdata/rds-egress-rule-timeout-http.json.golden

```diff
@@ -0,0 +1,91 @@
+{
+  "virtual_hosts": [
+   {
+    "name": "*.google.com:80",
+    "domains": [
+     "*.google.com",
+     "*.google.com:80"
+    ],
+    "routes": [
```

Comment:
Need to set OpaqueConfig for mixer. By defaults, it forwards attributes from source, which should not happen for out-of-cluster destinations.